### PR TITLE
Fix/recruit invitations

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

File: openreview/conference/templates/recruitReviewersProcess.js [hunk 1]
```diff
@@ -11,16 +13,29 @@ function() {
     if (note.content.response === 'Yes') {
       console.log('Invitation replied Yes');
       //if a user is in the declined group, remove them from that group and add them to the reviewers group
+      var email = {
+        groups: [note.content.user],
+        subject: '[' + SHORT_PHRASE + '] You have accepted the invitation',
+        message: 'This email is to confirm that you have accepted the invitation to be a ' + REVIEWER_NAME
```
Comment (melisabok) [inline openreview/conference/templates/recruitReviewersProcess.js]: I need a nice confirmation message here. Any help?

File: openreview/conference/templates/recruitReviewersProcess.js [hunk 2]
```diff
@@ -1,26 +1,42 @@
...
 
   if (hashKey === note.content.key) {
+    var commonMessage = 'If you want to change your choice please click on the accept or decline links in the invitation email.';
     if (note.content.response === 'Yes') {
       console.log('Invitation replied Yes');
       //if a user is in the declined group, remove them from that group and add them to the reviewers group
+      var email = {
+        groups: [note.content.user],
+        subject: '[' + SHORT_PHRASE + '] You have accepted the ' + REVIEWER_NAME + ' invitation',
+        message: 'This email is to confirm that you have accepted the invitation to be a ' + REVIEWER_NAME + '.\n\n' + commonMessage
```
Comment (zbialecki) [inline openreview/conference/templates/recruitReviewersProcess.js]: Text suggestion -> 'Thank you for accepting the invitation to be a REVIEWER_NAME for SHORT_PHRASE. The SHORT_PHRASE program chairs will be contacting you with more information regarding next steps soon. In the meantime, please add noreply@openreview.net to your email contacts to ensure that you receive all communications.

If you would like to change your decision, please click the Decline link in the previous invitation email.'

File: openreview/conference/templates/recruitReviewersProcess.js [hunk 3]
```diff
@@ -1,26 +1,42 @@
...
+      .then(result => or3client.or3request(or3client.mailUrl, email, 'POST', token))
       .then(result => done())
       .catch(error => done(error));
 
     } else if (note.content.response === 'No') {
       console.log('Invitation replied No');
+      var email = {
+        groups: [note.content.user],
+        subject: '[' + SHORT_PHRASE + '] You have declined the ' + REVIEWER_NAME + ' invitation',
+        message: 'This email is to confirm that you have declined the invitation to be a ' + REVIEWER_NAME + '.\n\n' + commonMessage
```
Comment (zbialecki) [inline openreview/conference/templates/recruitReviewersProcess.js]: Text suggestion -> 'You have declined the invitation to become a REVIEWER_NAME for SHORT_PHRASE. If you would like to change your decision, please click the Accept link in the previous invitation email.'

Maybe also say when the dealine for changing their decision is?

**Comment by melisabok** [comment]
@zbialecki  thanks for you feedback, I made all the changes, it looks really good now! I'm going to ask for more help with the communication 😄

**Comment by zbialecki** [comment]
Yeah, I'm happy to help out with this stuff!

File: openreview/conference/templates/recruitReviewersProcess.js [hunk 4]
```diff
@@ -1,26 +1,42 @@
...
+  var SHORT_PHRASE = '';
+  var REVIEWER_NAME = '';
   var REVIEWERS_ACCEPTED_ID = '';
   var REVIEWERS_DECLINED_ID = '';
   var HASH_SEED = '';
 
   var hashKey = or3client.createHash(note.content.user, HASH_SEED);
 
   if (hashKey === note.content.key) {
+    var commonMessage = 'If you want to change your choice please click on the accept or decline links in the invitation email.';
```
[the remark this answers is not in the document]
Comment (zbialecki) [reply openreview/conference/templates/recruitReviewersProcess.js]: I think this line can be removed

**Comment by melisabok** [comment]
This is ready to merge now.

**Pull request merged**